<launch>
    <!-- Parameters -->
    <param name="fixed_frame_id" type="str"    value="odom" />
    <param name="resolution"     type="double" value="0.1"  />
    <param name="max_range"      type="double" value="3.0"  />
    <param name="hit_prob"       type="double" value="0.8"  />
    <param name="miss_prob"      type="double" value="0.2"  />

    <!-- Run RViz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find realtime_occupancy_mapping)/rviz/example.rviz" />

    <!-- Run real-time occupancy mapping node -->
    <node pkg="realtime_occupancy_mapping" type="realtime_occupancy_mapping" name="realtime_occupancy_mapping">
        <remap from="/realtime_occupancy_mapping/pointcloud_in" to="/velodyne_points"/>
    </node>

</launch>